.github/workflows/Operator-CI.yaml

```diff
@@ -0,0 +1,67 @@
+name: Shard Operator - CI
+on:
+  pull_request:
+    paths:
+      - 'shard-operator/**'
```

Comment:
For now it is fine because it is a CI specific for the operator but I expect we will add also some end2end smoke test too